{"sections":[],"identifier":{"url":"doc:\/\/ATProtoKit\/documentation\/ATProtoKit\/ATAPIError\/unknown(message:)","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/ATProtoKit\/documentation\/ATProtoKit","doc:\/\/ATProtoKit\/documentation\/ATProtoKit\/ATAPIError"]]},"abstract":[{"type":"text","text":"Represents an unknown error with an associated message."}],"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unknown","kind":"identifier"},{"kind":"text","text":"("},{"text":"message","kind":"externalParam"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?)"}],"platforms":["macOS"],"languages":["swift"]}]},{"kind":"parameters","parameters":[{"name":"message","content":[{"inlineContent":[{"text":"The message received along side the error.","type":"text"}],"type":"paragraph"}]}]}],"variants":[{"paths":["\/documentation\/atprotokit\/atapierror\/unknown(message:)"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"symbolKind":"case","modules":[{"name":"ATProtoKit"}],"roleHeading":"Case","title":"ATAPIError.unknown(message:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unknown"},{"kind":"text","text":"("},{"kind":"externalParam","text":"message"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?)","kind":"text"}],"externalID":"s:10ATProtoKit10ATAPIErrorO7unknownyACSSSg_tcACmF","role":"symbol"},"kind":"symbol","schemaVersion":{"patch":0,"major":0,"minor":3},"references":{"doc://ATProtoKit/documentation/ATProtoKit":{"url":"\/documentation\/atprotokit","type":"topic","role":"collection","title":"ATProtoKit","abstract":[],"identifier":"doc:\/\/ATProtoKit\/documentation\/ATProtoKit","kind":"symbol"},"doc://ATProtoKit/documentation/ATProtoKit/ATAPIError":{"abstract":[{"type":"text","text":"The base exception class for ATProtoKitâ€™s API requests."}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ATAPIError"}],"type":"topic","kind":"symbol","title":"ATAPIError","url":"\/documentation\/atprotokit\/atapierror","identifier":"doc:\/\/ATProtoKit\/documentation\/ATProtoKit\/ATAPIError","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"ATAPIError"}]},"doc://ATProtoKit/documentation/ATProtoKit/ATAPIError/unknown(message:)":{"url":"\/documentation\/atprotokit\/atapierror\/unknown(message:)","role":"symbol","identifier":"doc:\/\/ATProtoKit\/documentation\/ATProtoKit\/ATAPIError\/unknown(message:)","title":"ATAPIError.unknown(message:)","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unknown"},{"text":"(","kind":"text"},{"text":"message","kind":"externalParam"},{"kind":"text","text":": "},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"kind":"text","text":"?)"}],"type":"topic","abstract":[{"text":"Represents an unknown error with an associated message.","type":"text"}],"kind":"symbol"}}}